moduleframework/mtf_scheduler.py

```diff
@@ -209,16 +217,27 @@ def __init__(self, args, unknown):
                 STATIC_LINTERS=common.conf["generic"]["static_tests"]))
         self.args = args
 
-        for param in unknown:
-            # take care of this, see tags for safe/unsafe:
-            # http://avocado-framework.readthedocs.io/en/52.0/WritingTests.html#categorizing-tests
-            testlist = glob.glob(param)
-            if testlist:
-                # this is list of tests in local file
+        # parse unknow options and try to find what parameter is test
+        while unknown:
```

Comment:
Why not used argparse?

Reply:
this behaviour was not change and it have sense to pass all unknown params to  avocado in case not known, like (--show-job-log) then it is possible to use mtf --show-job-log instread of create own param with same name and pass them to avocado.
create full argparser for each avocado param does not make sense. it is too complicated and avocado has plugin structure